plasmapy/utils/decorators/validators.py

```diff
@@ -540,9 +540,9 @@ def foo(arg1):
 
 def get_attributes_not_provided(
     self,
-    expected_attributes=None,
-    both_or_either_attributes=None,
-    mutually_exclusive_attributes=None,
+    expected_attributes: List[str] = None,
+    both_or_either_attributes: List[Iterable[str]] = None,
+    mutually_exclusive_attributes: List[Iterable[str]] = None,
```

Comment:
Technically - and purely technically since we aren't `mypy` compliant yet but I'm hoping to improve that soon - these should all be `Optional[List[str]]`, because `None` sure don't look like no list of strings to me 😜. But I'll be fixing those soon, anyway.

Reply:
Oh duh. Good catch